utils/src/main/java/com/cloud/utils/net/MacAddress.java

```diff
@@ -31,9 +31,9 @@
 
 import org.apache.log4j.Logger;
 
-import com.cloud.utils.NumbersUtil;
-
 /**
+ * This class retrieves the (first) MAC address for the machine is it is loaded on and stores it statically for retrieval.
+ * It can also be used for formatting MAC addresses.
  * copied from the public domain utility from John Burkard.
  * @author <a href="mailto:jb@eaio.com">Johann Burkard</a>
```

Comment:
When I see a piece of code like this I always ask myself if we should keep or delete authors references. What do you think of removing it?
In my humble opinion, we should clean lines such as this `@author`.